LLama/Batched/BatchedExecutor.cs

```diff
@@ -16,7 +15,12 @@ public sealed class BatchedExecutor
     : IDisposable
 {
     private int _nextSequenceId;
-    private readonly List<IBatch> _batchQueue = [ ];
+    private readonly List<IBatch> _batchQueue = [];
```

Comment:
This has introduced a lot of complexity to do with managing the queue and the cached token count into `BatchedExecutor`, it could probably be factored out into other classes in the future.